dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteFilesPresenter.java

```diff
@@ -52,10 +53,18 @@ public void initialize(final URL location, final ResourceBundle resources) {
         try {
             deleteButton.setDisable(true);
             if (ds3PanelPresenter.getDs3TreeTableView() != null) {
-                final ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+                ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3PanelPresenter.getDs3TreeTableView().getSelectionModel().getSelectedItems();
+                if (null == selectedPanelItems || selectedPanelItems.size() == 0) {
+                    selectedPanelItems = FXCollections.observableArrayList();
+                    selectedPanelItems.add(ds3PanelPresenter.getDs3TreeTableView().getRoot());
+                }
                 changeLabelText(selectedPanelItems);
             } else if (ds3TreeTablePresenter.ds3TreeTable != null) {
-                final ObservableList<TreeItem<Ds3TreeTableValue>> selectedMenuItems = ds3TreeTablePresenter.ds3TreeTable.getSelectionModel().getSelectedItems();
+                ObservableList<TreeItem<Ds3TreeTableValue>> selectedMenuItems = ds3TreeTablePresenter.ds3TreeTable.getSelectionModel().getSelectedItems();
+                if (null == selectedMenuItems || selectedMenuItems.size() == 0) {
```

Comment:
I believe we have some `Guard.isNullOrEmpty` methods which can be used to perform this check